internal/cli/up.go

```diff
@@ -62,12 +62,13 @@ func (c *upCmd) run(args []string) error {
 		return err
 	}
 
-	req := proto.CreateServiceRequest{Service: service, Watch: c.watch, LogLevel: proto.LogLevel(logLevel())}
-
-	err = dCli.CreateService(ctx, req)
-	s, ok := status.FromError(err)
-	if ok && s.Code() == codes.Unknown {
-		return errors.New(s.Message())
+	for i := range service {
+		req := proto.CreateServiceRequest{Service: service[i], Watch: c.watch, LogLevel: proto.LogLevel(logLevel())}
```

Comment:
Do we think it would be more efficient to change the RPC so that the _server_ is responsible for handling multiple services? It might be more efficient since then the client would only have to send one RPC, but the difference is probably negligible.

Reply:
Hmm. I don't know the answer to this question. Also wouldn't really know how to implement.

Reply:
No worries, I think this is fine for now. We can always make it more efficient later.

Reply:
yeah, efficiency aside, we need to do this on the server side, since the rpc won't return with `--watch`

Reply:
Do I leave it :sweat_smile: